Packs/Code42/ReleaseNotes/6_0_0.json

```diff
@@ -0,0 +1,4 @@
+{
+    "breakingChanges": true,
+    "breakingChangesNotes": null
```

Comment:
You marked breaking changes but didn’t include a reason. Please update breakingChangesNotes with the explanation.

Reply:
Per the documentation (https://xsoar.pan.dev/docs/documentation/release-notes#breaking-changes-version) I thought that if this was left null, the version release notes (from the markdown file) would be presented. Is this incorrect? I can certainly update but it will be the same information.

Reply:
You are right.